src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -34,8 +34,9 @@
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
                         <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                                    Name="ContextMenuPanel">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding ElementName=ContextMenuPanel, Path=ActualWidth}" />
```

Comment:
There is no any limitation why we can't use `Grid` instead of `StackPanel`. But in any case we will have to limit width of `InCanvasSearch` by binding it to parent element's width. It's because `InCanvasSearchControl` has inside horizontal scroll viewer. If we don't limit width, then this scroll viewer will take the whole width, that he wants.